lib/streamlit/logger.py

```diff
@@ -109,7 +109,7 @@ def get_logger(name):
     Parameters
     ----------
     name : str
-        The name of the logger to use. You should just pass in __name__.
```

Comment:
Hm, is this public-facing? I feel like internally we should continue using these words to convey nuance.

(Ditto for the usages below)